lib/build-utilities/migrate-initializers.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const path = require('path');
+const existsSync = require('exists-sync');
+const fs = require('fs-extra');
+
+const fastbootInitializerTypes = [ 'initializers', 'instance-initializers'];
+const FASTBOOT_DIR = 'fastboot';
```

Comment:
Given the description of the PR I was expecting the new FastBoot folder to be named `fastboot-app` rather than `fastboot`, but this might be just some misunderstanding?